src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="body">
+    <main class="calendar calendar--mounth--mounth29">
+      <div class="day day--1 calendar__days day--day--sunday"></div>
+      <div class="day day--2 calendar__days"></div>
```

Comment:
It will be better for supporting code if use less classes. You can implement modifiers only in block "calendar" and don't add classes for elements exept "day"